be/src/fs/credential/cloud_configuration_factory.h

```diff
@@ -29,6 +29,7 @@ static const std::string AWS_S3_USE_AWS_SDK_DEFAULT_BEHAVIOR = "aws.s3.use_aws_s
 static const std::string AWS_S3_USE_INSTANCE_PROFILE = "aws.s3.use_instance_profile";
 static const std::string AWS_S3_ACCESS_KEY = "aws.s3.access_key";
 static const std::string AWS_S3_SECRET_KEY = "aws.s3.secret_key";
+static const std::string AWS_S3_SESSION_TOKEN = "aws.s3.session_token";
```

Comment:
add some comments about this tabular properties.

Reply:
session token is not only for tabular, it's aws temporary ak/sk.
That means we can use below grammar now:
```sql
"aws.s3.access_key"="xxx",
"aws.s3.secret_key"="yyy",
"aws.s3.session_token"="xxx"
```